app/assets/stylesheets/css3/_flex-box.scss

```diff
@@ -2,58 +2,78 @@
 
 // Custom shorthand notation for flexbox
 @mixin box($orient: inline-axis, $pack: start, $align: stretch) {
+  @include _bourbon-deprecate-for-prefixing("box");
+
   @include display-box;
   @include box-orient($orient);
   @include box-pack($pack);
   @include box-align($align);
 }
 
 @mixin display-box {
+  @include _bourbon-deprecate-for-prefixing("display-box");
+
   display: -webkit-box;
   display: -moz-box;
   display: -ms-flexbox; // IE 10
   display: box;
 }
 
 @mixin box-orient($orient: inline-axis) {
+  @include _bourbon-deprecate-for-prefixing("box-orient");
+
 // horizontal|vertical|inline-axis|block-axis|inherit
```

Comment:
Yes, these little inconsistencies drive me bonkers. But since this is all old code that has already been removed in 5.0, I try and not touch it so to create clean diffs and commits, and also not accidentally introduce bugs.

5.0 is super duper consistent and clean, though 😉